template/src/components/RTMConfigure.tsx

```diff
@@ -28,16 +29,38 @@ export enum UserType {
   ScreenShare,
 }
 
+const adjustUID = (number: number) => {
+  if (number < 0) {
+    number = 0xffffffff + number + 1;
+  }
+  return number;
+};
+
+const stringifyPayload = (source: mSourceType, type: mType, msg: string) => {
```

Comment:
use destructring when more than one args, be verbose with the arg names 
```suggestion
const stringifyPayload = ({messageSource: mSourceType, type: messageType, message: string}) => {
```
or

```suggestion
const stringifyPayload = ({msgSource: mSourceType, type: msgType, msg: string}) => {
```